source/styleEditor.js

```diff
@@ -1,3 +1,14 @@
 $(document).ready(function(){
-  // your code goes here.
+  var changeCss = function() {
```

Comment:
a bit wonky with the values being referenced in different places. a cleaner solution would 

```
var changeCss = function(event){
    let cssSelector = $('[name=selector]').val();
    let cssProperty = $('[name=property]').val();
    let cssValue    = $('[name=value]').val();
    $(cssSelector).css(cssProperty, cssValue);
   
    //Want to include this as well so that we prevent the browser from trying to submit the form
    event.preventDefault();
 }
```